test/loadTest/use_of_epoll/use_epoll.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+"""
+ Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+
+ This file is part of Orion Context Broker.
+
+ Orion Context Broker is free software: you can redistribute it and/or
+ modify it under the terms of the GNU Affero General Public License as
+ published by the Free Software Foundation, either version 3 of the
+ License, or (at your option) any later version.
+
+ Orion Context Broker is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+
+ For those usages not covered by this license please contact with
+ iot_support at tid dot es
+"""
+__author__ = 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+import sys
+import requests
+import random
+import time
+import datetime
+
+
+class Epoll:
+    """
+    Verify the epoll() instead of select(). See https://jirapdi.tid.es/browse/DM-2375 issue
+    Tests procedure:
+    - create 5000 subscription with subject.entities.idPattern:. *
+    - verify/modify the notification listener with a delay of 10 minutes before answering.
+    - modify the ContextBroker config with: -httpTimeout 600000 and restart it.
+    - modify a entity each 3 minutes, that it triggers all subscriptions.
```

Comment:
You don't need to do this each 3 minutes... Just doing it at the very beginning is ok.

Maybe a step to check how many ESTABLISH connection has the broker will be a good adition after this step.